tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -62,85 +140,128 @@ def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
         node_one = worker_nodes_list[0]
         node_two = worker_nodes_list[1]
 
-        # Create a PVC
-        accessmode = constants.ACCESS_MODE_RWX
-        if self.interface == constants.CEPHBLOCKPOOL:
-            accessmode = constants.ACCESS_MODE_RWO
-        pvc_obj = pvc_factory(
-            interface=self.interface,
-            access_mode=accessmode,
-            status=constants.STATUS_BOUND,
-            size="15",
-        )
-
-        # Create a pod on one node
-        logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_one}")
-
-        helpers.pull_images(constants.PERF_IMAGE)
-        pod_obj1 = helpers.create_pod(
-            interface_type=self.interface,
-            pvc_name=pvc_obj.name,
-            namespace=pvc_obj.namespace,
-            node_name=node_one,
-            pod_dict_path=constants.PERF_POD_YAML,
-        )
+        time_measures = []
+        for sample_index in range(1, samples_num + 1):
+            # Create a PVC
+            accessmode = constants.ACCESS_MODE_RWX
+            if self.interface == constants.CEPHBLOCKPOOL:
+                accessmode = constants.ACCESS_MODE_RWO
+            pvc_obj = pvc_factory(
+                interface=self.interface,
+                access_mode=accessmode,
+                status=constants.STATUS_BOUND,
+                size="15",
+            )
```

Comment:
it is good to wrap it with try...except and verify the PVC actually created

Reply:
Added try... except with exception if the PVC sample was not created.